phpunit.xml.dist

```diff
@@ -2,10 +2,10 @@
 	bootstrap="tests/php/bootstrap.php"
 	backupGlobals="false"
 	colors="true"
-	convertDeprecationsToExceptions="true"
-	convertErrorsToExceptions="true"
-	convertNoticesToExceptions="true"
-	convertWarningsToExceptions="true"
+	convertDeprecationsToExceptions="false"
+	convertErrorsToExceptions="false"
+	convertNoticesToExceptions="false"
+	convertWarningsToExceptions="false"
```

Comment:
It can be reverted now?

Reply:
Not yet. See https://github.com/ampproject/amp-wp/pull/7225#discussion_r977290405